src/code/z_message.c

```diff
@@ -4429,7 +4429,7 @@ void Message_Update(PlayState* play) {
 
                     if (msgCtx->textId != 0x2061 && msgCtx->textId != 0x2025 && msgCtx->textId != 0x208C &&
                         ((msgCtx->textId < 0x88D || msgCtx->textId >= 0x893) || msgCtx->choiceIndex != 0) &&
-                        (msgCtx->textId != 0x3055 && gSaveContext.save.cutsceneIndex < 0xFFF0)) {
+                        (msgCtx->textId != 0x3055 && gSaveContext.save.cutsceneIndex < CS_INDEX_0)) {
```

Comment:
```suggestion
                        (msgCtx->textId != 0x3055 && !(gSaveContext.save.cutsceneIndex >= CS_INDEX_0))) {
```

Does this match?

Reply:
yes, but this would be for researching making a macro which is out of scope here